build.proj

```diff
@@ -62,26 +62,27 @@
                             Condition=" '$(Scope)' == 'AzureStorage' "/>
     <SetupSln Include=".\setup\azurepowershell.sln" />
     <StaticAnalysis Include=".\tools\StaticAnalysis\StaticAnalysis.sln" />
+    <LocalBuildTasks Include="$(LibraryToolsFolder)\BuildPackagesTask\Microsoft.Azure.Build.Tasks.sln" />
```

Comment:
isn't this property already defined elsewhere?  Seems like it needs to be for us to build the other local build tasks